src/Raven.Studio/typescript/viewmodels/database/query/querySyntax.ts

```diff
@@ -82,14 +82,24 @@ select {
     OrderedAt: i.OrderedAt
 }`,
             html:
-`<span class="token keyword">from index</span> <span class="token string">'Orders/Totals'</span> <span class="token keyword">as</span> i
+`<span class="token keyword">from index</span> <span class="token string">&quot;Orders/Totals&quot;</span> <span class="token keyword">as</span> i
 <span class="token keyword">where</span> i.Total <span class="token operator">></span> <span class="token number">10000</span>
 <span class="token keyword">load</span> i.Company <span class="token keyword">as</span> c
 <span class="token keyword">select</span> <span class="token punctuation">{</span>
     Name: c.Name,
     Region: c.Address.Region,
     OrderedAt: i.OrderedAt
 <span class="token punctuation">}</span>`
+        },
+        {
+            title: "Vector search",
+            text:
+`from Products
+where vector.search(embedding.text(Name), "italian", 0.8)`,
```

Comment:
Maybe let's change it into
```from Products where vector.search(embedding.text(Name), "italian food")```

Reply:
@maciejaszyk 
should I move this fix (and this PR) to `release/v7.0` ? 
or leave here ?

Reply:
this should go to `release/v7.0`